src/components/GlobalPreferences/Notifications/SubscriptionsTable.js

```diff
@@ -92,20 +92,25 @@ const SubscriptionsTable = React.memo(function SubscriptionsTable({
     setDeleteModalOpened(false)
   }, [])
 
-  const organizations = Array.from(
-    new Set(subscriptions.map(subscription => subscription.ensName))
-  )
-  const subscriptionApps = Array.from(
-    new Set(subscriptions.map(subscription => subscription.appName))
-  )
-  const events = Array.from(
-    new Set(subscriptions.map(subscription => subscription.eventName))
-  )
+  const organizations = [
+    'All',
+    ...new Set(subscriptions.map(subscription => subscription.ensName)),
+  ]
+  const subscriptionApps = [
+    'All',
+    ...new Set(subscriptions.map(subscription => subscription.appName)),
+  ]
+  const events = [
+    'All',
+    ...new Set(subscriptions.map(subscription => subscription.eventName)),
+  ]
   const filteredSubscriptions = filterSubscriptions({
     subscriptions,
-    event: events[selectedEvent],
-    appName: subscriptionApps[selectedApp],
-    organization: organizations[selectedOrganization],
+    event: selectedEvent > 0 ? events[selectedEvent] : null,
```

Comment:
In addition to doing this, we should also default the `selectedEvent`, `selectedApp` and `selectedOrganization` state to be -1 (when we select 'All'), so that it deactivates the filter.

Reply:
@sohkai 
Not sure I understood but do you mean selecting `-1`, i.e. the dropdown label when `All` is selected? 

If so, I've pushed a commit for that.

Reply:
Yep, that's what he meant. We' ve been doing it this way: when the user selects `All`, we actually set the index to `-1` (as if none was selected). When none, is selected then apply no filter and the filter does not show the blue outline for a (non `-1`) selected option.
Hope this helps clarify.